starknet_py/net/client_models.py

```diff
@@ -3,24 +3,8 @@
 from enum import Enum
 from typing import Any, Dict, Iterable, List, Optional, Union
 
-from starkware.starknet.services.api.gateway.transaction import AccountTransaction as AT
-from starkware.starknet.services.api.gateway.transaction import ContractClass as CD
-from starkware.starknet.services.api.gateway.transaction import Declare as DCL
-from starkware.starknet.services.api.gateway.transaction import DeployAccount as DAC
-from starkware.starknet.services.api.gateway.transaction import InvokeFunction as IF
-from starkware.starknet.services.api.gateway.transaction import Transaction as T
 from typing_extensions import Literal
 
-from starknet_py.utils.docs import as_our_module
-
-Invoke = InvokeFunction = as_our_module(IF)
-StarknetTransaction = as_our_module(T)
-AccountTransaction = as_our_module(AT)
-ContractClass = as_our_module(CD)
-Declare = as_our_module(DCL)
-DeployAccount = as_our_module(DAC)
-
-
```

Comment:
This is breaking I think. We should include it in migration guide.

Reply:
#659